package.json

```diff
@@ -52,10 +52,6 @@
   "dependencies": {},
   "devDependencies": {
     "ava": "^0.15.2",
-    "babel": "^6.5.2",
-    "babel-cli": "^6.18.0",
-    "babel-core": "^6.21.0",
-    "babel-preset-es2015": "^6.18.0",
```

Comment:
@dav-m85 these are why the tests fails in node 4.x , sorry , need to add them back, they need to be explictely added